themes/jb/layouts/sponsors/list.html

```diff
@@ -0,0 +1,25 @@
+{{ define "main" }}
+    <section class="spnosor-list px-4 mb-5">
+        <div class="container">
+            <div class="column is-full content">
+                <h1 class="title">Sponsors</h1>
+                <h4>We have many sponsors we love who support each show, and we hope you'll enjoy them too:</h4>
```

Comment:
> There is a "Jupiter Broadcasting" section that is omitted from the PR version that would be great to integrate.
> 
> ```
> Jupiter Broadcasting
> Linode - receive a $100 60-day credit towards your new account.
> [Linode.com/jupiter](https://linode.com/jupiter)
> ```
> 
> ![image](https://user-images.githubusercontent.com/7018928/195003758-35378879-c97e-408b-b56e-f3bf7e538853.png)
> 
> This section is quite helpful for network-wide sponsorship links, and doesn't really appear in episodes regularly.
> 
> Having a way of adding non-automated links occasionally would be helpful aswell, for instance as described in #458

For :arrow_up:, we could probably still use the `_index.md` to handle the network wide stuff (i.e. jupiter.party). So we could put it right below this line and and right before the range. Just add the `{{ .Content }}` and it should just work (we'll have to cleanup the `_index.md` file though to only use the necessary JB wide stuff.

I think the #458 might be a bit difficult if we end up listing 2 Self-Hosted sections or something, but if it's just all lumped under a JB network wide section I think that'd be fine.

Reply:
I added the sponsor list to the metadata so it uses the same "interface" as the episodes.

Reply:
Awesome, nice thinking! I hadn't considered that but it's probably the best route to use :grin: 

Just as a note (mainly for everyone else), this means the JB wide sponsorships will have to be manually updated by doing the following:
1. If there's a network wide sponsor that should be shown then there needs to be a similarly named file for the corresponding sponsor
   - for example [jupiter party](https://jupiter.party) would be named: `jupiter.party-jb` (which is `<sponsors_domain>-<corrisponding_show_slug>`
2. Then it would have the same content/front matter as what @ChanceM did [here](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/451/files#diff-31db6f7f953069daf34794d2665d45ea58cd19d05854b777565922e973dfee3e)
3. Lastly it's filename will need to be added to the front matter array of sponsors in the `content/sponsors/_index.md` file